invite/forms.py

```diff
@@ -115,6 +117,19 @@ def clean(self, *args, **kwargs):
         self.clean_email()
         return self.cleaned_data
 
+class BaseInviteItemFormSet(forms.BaseFormSet):
+    def clean(self):
+        """
+        Validates that there are no duplicate email addresses across forms.
+        """
+        super().clean()
+        emails = []
+        for form in self.forms:
+            email = form.cleaned_data.get('email')
+            if email:
+                if email in emails:
+                    raise ValidationError(f"{email} already belongs to another invitation in the current form")
```

Comment:
Could combine the conditions and use single quotes for consistency.
```suggestion
            if email and email in emails:
                raise ValidationError(f'{email} already belongs to another invitation in the current form')
```